### ISSUE TITLE
File Input

### ISSUE THREAD

**Issue by aliqasemzadeh** [wireui/wireui#44]
Hi
I think we need file input. multiple.

**Comment by Tiagospem** [comment]
The inputs.currency has a problem, when I use 
thousands="." decimal="," precision="2"  it shows on view  "R$1.225,50" (1225.50)
so when I get 1225.50 from DB and pass to wire:model, it shows "R$125.050",

**Comment by sw1tchmx** [comment]
how to change the type of input? i want to change it to "number" or "tel"

**Comment by ph7jack** [comment]
@sw1tchmx 
> how to change the type of input? i want to change it to "number" or "tel"

`<x-input type="number" />`

Available types
https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/input

**Comment by aliqasemzadeh** [comment]
@PH7-Jack 
Did you make file input component?

**Comment by ph7jack** [comment]
@alighasemzadeh No, I'm developing the new release 2.x to be able to create new components

**Comment by Uhasith** [comment]
Voted for this !
